src/app/base/components/bricks-integrator/helper/code-parser.ts

```diff
@@ -47,7 +49,7 @@ export class BricksCodeParser {
     }
 
     public replaceVariables() {
-        let replacedCode = this.baseCode;
+        let replacedCode = this.baseCode.replace(getPythonFunctionName(this.baseCode), this.functionName);
```

Comment:
not sure if this approach work reliably since only the functionname is used to replace. This means if the function name is e.g. provided in a previous comment this will run into issues. I think we should try to find the line index of the function and change the value at that index to be sure.